environment.yml

```
@@ -107,6 +107,7 @@ dependencies:
 
   # Packages specific to cryo work
   - icepyx==0.7.0
+  - sliderule==3.5.0
```

`sliderule` uses `ipyleaflet` 0.17.3 for some visualization (REMA/ArcticDEM). might be good to bump that up as well.

---

Yep, and we're moving towards compatible release pins `~=` now for non CalVer packages per #46.
```suggestion
  - sliderule~=3.5.0
```